src/views/Account.vue

```diff
@@ -37,6 +37,47 @@
 					</div>
 				</div>
 			</div>
+
+			<div class="user sinatures">
+				<h1>{{ t('libresign', 'Your signatures') }}</h1>
+				<div class="signature-fav">
+					<header>
+						<h2>{{ t('libresign', 'Signature') }}</h2>
+						<div v-if="haveSignature" class="icon icon-rename" @click="editSignatures" />
+					</header>
+
+					<img v-if="haveSignature" :src="sinatures">
+					<div v-else class="no-signatures" @click="editSignatures">
+						<span>
+							{{ t('libresign', 'No signature, click here to create a new') }}
+						</span>
+					</div>
+				</div>
+				<div class="signature-fav">
+					<header>
+						<h2>{{ t('libresign', 'Initials') }}</h2>
+						<div v-if="haveInitials" class="icon icon-rename" @click="editInitials" />
+					</header>
+					<img v-if="haveInitials" :src="sinatures">
```

Comment:
Is a good practice implement alt text focusing on accessibility

Reply:
This PR is in Draft, it is incomplete.